quipucords/api/scan/model.py

```diff
@@ -130,6 +131,11 @@ def get_default_forks():
         """Create the default number of forks."""
         return DEFAULT_MAX_CONCURRENCY
 
+    @staticmethod
+    def get_max_forks():
+        """Create the maximum number of forks."""
+        return UPPER_MAX_CONCURRENCY
```

Comment:
I understand you are following the pattern established previously for `get_default_forks` but do you see an advantage in creating a static method on a class that returns a constant instead of simply creating a constant in it?

Not saying we should change it, just trying to see if there's an advantage I'm not seeing here.

Reply:
Still using the method, I will update to use a Scan constant.